build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -269,6 +269,21 @@ service ContentAddressableStorage {
   }
 }
 
+// The Capabilities service may be used by remote execution clients to query
+// various server properties, in order to self-configure or return meaningful
+// error messages.
+// The query may include a particular `instance_name`, in which case the values
+// returned will pertain to that instance.
+service Capabilities {
+  // GetCapabilities returns the server capabilities configuration.
+  rpc GetCapabilities(GetCapabilitiesRequest)
+    returns (GetCapabilitiesResponse) {
```

Comment:
GET requests should return a resource message, not a separate response message (https://cloud.google.com/apis/design/standard_methods).

Reply:
Done.

Reply:
Wait, now that I think about it -- should it always be GetX returns X? If so, I'm confused as to how to name this -- the service can't have the same name as the message, right?